modules/bookIsEmpty.js

```diff
@@ -0,0 +1,10 @@
+const StoredData = JSON.parse(localStorage.getItem('books')).length;
+
+const bookIsEmpty = () => {
+  if (StoredData === 0) {
```

Comment:
- Hey great job  using the es6 syntax, however your code is not running due to the `.length` you've added in this line of code `const StoredData = JSON.parse(localStorage.getItem('books')).length;`. Since you're adding the length to check if the array is empty, kindly move the `.length` to the 
`if (StoredData.length === 0) {` to get the length property of the array to get the expected result.